src/utils/filterCountries-api.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable consistent-return */
+import axios from 'axios';
+
+const transferDate = (date) => date.toISOString().split('T')[0];
+
+const today = new Date();
+let todayDate = transferDate(today);
+
+const yesterday = new Date();
+yesterday.setDate(today.getDate() - 1);
+const yesterDate = transferDate(yesterday);
+
+const config = {
+  method: 'get',
+  url: `https://api.covid19tracking.narrativa.com/api/${todayDate}`,
+};
+
+const fetchCountries = async (name) => {
+  const res = await axios(config);
+  if (res.status === 200) {
+    const contriesData = await res.data.dates[`${todayDate}`].countries;
+    const allCountries = Object.keys(contriesData).map((key) => contriesData[key]);
+    const filteredCountry = allCountries.filter((obj) => obj.id.includes(name));
+
+    // Non camelCase are needed here since they're used by the Narrativa API.
+
+    /* eslint-disable camelcase */
```

Comment:
- [ ] please kindly avoid using several eslint-disable comments to get around ESlint errors. these ESlint errors are at best showing that you are not adhering to some coding standards or at worst highlighting a possible underlying issue with your code. please kindly address these issues.

Reply:
- [ ] You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean as recommended.